### PR TITLE
Literature model

### PR DESCRIPTION
**Pull request by danich1**
This PR implements and compares the CoCoScore text mining system against my current system. I used hetionet v1 edges for the comparison, where the positives are edges already established in hetionet and the negatives are edges not in hetionet v1. There might be a might be a confounding issue for precision-recall performance, since hetionet v1 is not a fully complete network (some non-zero edges might actually be true edges but are considered negative in this context).

### CODE REVIEW

File: compound_disease/compound_treats_disease/literature_models/coco_score/coco_score.py [hunk 1]
```diff
@@ -0,0 +1,376 @@
+
+# coding: utf-8
+
+# # CoCoScore Implementation
+
+# This notebook consists of implementing the [CoCoScore](https://www.biorxiv.org/content/10.1101/444398v1) literature model for comparison.
```
Comment (ajlee21) [inline compound_disease/compound_treats_disease/literature_models/coco_score/coco_score.py]: Might have a sentence that describes the metric in addition to the hyperlink

File: compound_disease/compound_treats_disease/literature_models/coco_score/coco_score.py [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+
+
+all_embedded_cd_df = pd.read_table("../../word_vector_experiment/results/all_embedded_cd_sentences.tsv.xz")
+all_embedded_cd_df.head(2)
+
+
+# In[11]:
+
+
+word_vectors = pd.read_csv(
```
Comment (ajlee21) [inline compound_disease/compound_treats_disease/literature_models/coco_score/coco_score.py:126]: Why is the first row blank?

File: compound_disease/compound_treats_disease/literature_models/coco_score/coco_score.py [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+
+
+# In[25]:
+
+
+precision, recall, _ = precision_recall_curve(
+    total_dev_sentences_df.curated_ctd,
+    pd.read_table('dev_predictions.tsv', header=None, sep=" ")[1]
+)
+auc(recall, precision)
```
Comment (ajlee21) [inline compound_disease/compound_treats_disease/literature_models/coco_score/coco_score.py:248]: So these curves are how accurately you were able to predict an edge in hetionet?

**Review by ajlee21 — APPROVED** [review]
Looks good.  Just a few comments that you can feel free to get to after your manuscript is due, as discussed.

So for the .png file, you're assuming CoCo score is the baseline that you want to compare you're relationships to?

File: compound_disease/compound_treats_disease/literature_models/coco_score/coco_score.py (hunk 2 above)
Comment (danich1) [reply compound_disease/compound_treats_disease/literature_models/coco_score/coco_score.py:126]: Good catch that is a typo that I'll make sure to fix.

File: compound_disease/compound_treats_disease/literature_models/coco_score/coco_score.py (hunk 3 above)
Comment (danich1) [reply compound_disease/compound_treats_disease/literature_models/coco_score/coco_score.py:248]: Exactly. These curves show the varying recall and precision thresholds for each cutoff score. The better the model the higher the area under the curve is.

**Comment by danich1** [comment]
> So for the .png file, you're assuming CoCoScore is the baseline that you want to compare you're relationships to?

The .png file compares our model with the CoCoScore literature model. For this particular instance CoCoScore is our "baseline" model. The goal here is to show that our method performs just as good as a literature model.

**Pull request merged**